README.md

```diff
@@ -50,13 +50,13 @@ The `users` directory contains a list of files, each representing a host on mit-
 
 Available fields:
 
-* copyright (required)
-* URL
-* email
-* format
-* gravatar
-* theme
-* license
+- copyright (required)
+- URL
+- email
+- format
+- gravatar
+- theme
+- license
```

Comment:
Don't make unrelated changed

Reply:
I have a formator install, it made some changes on save.

Reply:
This still means you'll have to revert these changes.